index.js

```diff
@@ -0,0 +1,44 @@
+
+
+// First exercise --------------------
+const countMovesUp = inputValue => inputValue.match(/\(/g).length
+
+const countMovesDown = inputValue => inputValue.match(/\)/g).length
```

Comment:
This `match` returns `null` if didn't find the pattern, so when `length` is called we'll got an exception. Maybe you should verify the result of `match` before call `length`.

Something like that:
`inputValue.match(/\)/g) ? inputValue.match(/\)/g).length : 0`